drgn/cli.py

```diff
@@ -209,6 +209,21 @@ def _main() -> None:
         help="don't load any debugging symbols that were not explicitly added with -s",
     )
 
+    parser.add_argument(
+        "-a",
+        "--architecture",
+        metavar="ARCH",
+        choices=[a.name for a in drgn.Architecture]
+        + [a.name.lower() for a in drgn.Architecture],
+        help="set the program architecture, in case it can't be auto-detected",
+    )
+    parser.add_argument(
+        "-i",
+        "--vmcoreinfo",
```

Comment:
Let's add this for consistency with the other options taking paths:
```suggestion
        metavar="PATH",
```